src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/resendtogaps/ResendToGapsMessageHandler.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.resendtogaps;
+
+import static uk.gov.hmcts.reform.sscs.ccd.domain.HearingRoute.LIST_ASSIST;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.HearingState.CANCEL_HEARING;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.model.hearings.HearingRequest;
+import uk.gov.hmcts.reform.sscs.service.servicebus.HearingMessagingServiceFactory;
+
+@Service
+@RequiredArgsConstructor
+public class ResendToGapsMessageHandler {
+
+    private final HearingMessagingServiceFactory hearingMessagingServiceFactory;
+
+    public boolean sendMessage(final String ccdCaseId) {
```

Comment:
Could you make this a bit more generic? So pass in the Route + State

Reply:
I think the API is cleaner this way.. can change the method name to `sendCancelMessage`?

Reply:
We could reuse this for other requests. I have something very similar in my PR - all that's changing is the route and state. This whole class could become a message handler, all callers would just need to provide the required details.